src/Cli/UpdateTerminusDocsCommands.php

```diff
@@ -0,0 +1,163 @@
+{
+    use ConfigAwareTrait;
+    use LoggerAwareTrait;
+
+    /**
+     * Update terminus commands and releases in documentation repo.
+     *
+     * @command terminus:update-docs
+     */
+    public function terminusUpdateDocs($options = [
+        'as' => 'default',
+        'update-commands' => true,
+        'update-releases' => true,
+        'terminus-release' => 'latest',
+        'github-repo' => 'pantheon-systems/documentation',
+        'terminus-repo' => 'pantheon-systems/terminus',
+        'base-branch' => 'main',
+        'branch-name-prefix' => 'docs-update-terminus-',
+        'commit-message' => 'Update terminus information.',
+        'pr-body' => '',
+        'pr-title' => '[UpdateTool - Terminus Information] Update commands and releases.',
+        'dry-run' => false,
+    ])
+    {
+        $api = $this->api($options['as']);
+        $updateCommands = $options['update-commands'];
+        $updateReleases = $options['update-releases'];
+        if (!$updateCommands && !$updateReleases) {
+            throw new \Exception("Either commands or releases must be updated.");
+        }
+
+        $terminusRepo = $options['terminus-repo'];
+        $terminusRelease = $options['terminus-release'];
+
+        $terminusDir = sys_get_temp_dir() . '/update-tool/terminus/';
+        if (!is_dir($terminusDir)) {
+            mkdir($terminusDir, 0755, true);
+        }
+
+        if ($terminusRelease === 'latest') {
+            $terminusRelease = $this->getLatestRelease($api, $terminusRepo);
+        }
+        $downloadUrl = 'https://github.com/pantheon-systems/terminus/releases/download/' . $terminusRelease . '/terminus.phar';
+
+        file_put_contents($terminusDir . 'terminus.phar', file_get_contents($downloadUrl));
+        if (!file_exists($terminusDir . 'terminus.phar')) {
+            throw new \Exception(sprintf('Failed to download terminus.phar from release %s.', $terminusRelease));
+        }
+        chmod($terminusDir . 'terminus.phar', 0755);
+
+        $githubRepo = $options['github-repo'];
+        $baseBranch = $options['base-branch'];
+
+        $branchNamePrefix = $options['branch-name-prefix'];
+        $branchName = $branchNamePrefix . date('Ymd');
+
+        $url = 'git@github.com:' . $githubRepo . '.git';
+        $remote = new Remote($url);
+        $dir = sys_get_temp_dir() . '/update-tool/' . $remote->project();
+        $workingCopy = WorkingCopy::cloneBranch($url, $dir, $baseBranch, $api);
+        $workingCopy->setLogger($this->logger);
+        $workingCopy->createBranch($branchName, $baseBranch, true);
+
+        if ($updateCommands) {
+            $this->logger->info('Updating Terminus commands...');
+            exec(sprintf("cd %s && %s/terminus.phar list --format=json > source/data/commands.json", $dir, $terminusDir), $output, $return);
+            if ($return != 0) {
+                throw new \Exception(sprintf('Failed to list terminus commands. Output: %s', implode("\n", $output)));
```

Comment:
Include `$return` or `$exitCode` value also for completeness?

Reply:
Done. thanks